ABCD/data_convert.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+from pathlib import Path
+
+import pandas as pd
+
+
+def get_args():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter, description=__doc__)
+    parser.add_argument('-t', '--tabular-data', type=Path, action='store', dest='tab_data', metavar='TAB_DATA_DIR',
+                        help='Path to directory with tabular data files in \'.txt\' format.')
+    parser.add_argument('-d', '--dictionaries', type=Path, action='store', dest='dicts', metavar='DICTS_DIR',
+                        default=Path('.'), help="Path to directory with data dictionaries in \'.csv\' format.")
+    parser.add_argument('-o', '--output-dir', type=Path, action='store', dest='outdir', metavar='OUTPUT_DIR',
+                        default=Path('.'), help="Destination directory for the \'.tsv\' formatted files.")
+    args = parser.parse_args()
+
+    return args.tab_data.resolve(), args.dicts.resolve(), args.outdir.resolve()
+
+
+def data_summary(data_with_no_dict):
+    print(f"A total of {len(data_with_no_dict)} data files don't have a corresponding data dictionary.")
+    print(data_with_no_dict)
```

Comment:
Yikes! Did this ever happen in testing?

Reply:
@Arshitha That's fishy and we need to get to the bottom of it, yeah.

Reply:
I'll add this as an issue.